sdk/storage/azure-storage-blob/HISTORY.md

```diff
@@ -5,7 +5,7 @@
 **New features**
 - Added support for the 2019-07-07 service version.
 - Added support for encryption scopes that that could be used to encrypt blob content. 
-- Added `get_managed_disk_page_range_diff` API which returns the list of valid page ranges diff between a snapshot and managed disk or another snapshot.
+- Added `managed_disk_diff` parameter to the `BlobClient.get_page_ranges` API which returns the list of valid page ranges diff between a snapshot and managed disk or another snapshot.
```

Comment:
I'm not sure if we've decided to merge managed_disk_diff with get_page_ranges. Let's discuss this with teammates from other SDKs.

Reply:
@xiafu-msft  - I have reverted the API change for now so we can discuss it separately without holding up the change for API versions. :)